ocd/src/collection/cores/m4.rs

```diff
@@ -206,8 +259,6 @@ impl Core for M4 {
             mi.write32(Dcrdr::ADDRESS, value).map_err(From::from);
         result?;
 
-        self.wait_for_core_register_transfer(mi)?;
```

Comment:
Any reason you removed this line?

Reply:
From my understanding of the ARMv6/ARMv7 architecture reference manual. `DHCSR.S_REGRDY` is only affected by `DCRSR` which means that it is unnecessary to check after writing `DCRDR`. But maybe this is fixing a quirk?